src/server.mjs

```diff
@@ -283,8 +283,30 @@ async function getChunkedData(req, res, next) {
 
     // We're always replacing the given tracks if we were able to find tracks from the bed file
     if (fetchedTracks) {
-      console.log("Using new fetched tracks", JSON.stringify(req.body.tracks));
-      req.body.tracks = fetchedTracks;
+      // Color Settings are retained from the initial request
+      // if newly fetched tracks have matching file names
+      // Store current colors and file names
+      const fileToColor = new Map();
```

Comment:
We're sure that we're not going to have multiple tracks with the same filename?

Can we not use the track key because we want to think about tracks being added/removed/reordered?